src/main/frontend/state.cljs

```diff
@@ -700,7 +702,15 @@
 
 (defn get-selection-blocks
   []
-  (util/sort-by-height (:selection/blocks @state)))
+  (:selection/blocks @state))
+
+(defn get-selection-block-ids
+  []
+  (->> (sub :selection/blocks)
+       (map #(when-let [id (dom/attr % "blockid")]
+               (medley/uuid id)))
```

Comment:
This is just `uuid` in cljs. If we're not writing a cljc file, I don't see why we'd depend on an external library for a core fn